docs/guides/debugging.md

```diff
@@ -922,3 +922,33 @@ latter, the following commands might be of help:
     scylla generate-object-graph # generates and visualizes an object graph
 
 Good luck, you are off the charted path here.
+
+#### Use python for inspecting objects and performing computations
+GDB is well integrated with Python, and besides using the predefined set of commands from `scylla-gdb.py`, you can also run an interactive prompt by typing `python-interactive` or `pi` in gdb console. That allows you to benefit from the rich set of helper classes and wrappers defined in `scylla-gdb.py`, e.g. in order to iterate over one of the supported collection types (vectors, small vectors, maps, tuples, etc.) and print only the interesting bits.
```

Comment:
This file uses line length of 80. I know it is minor thing but please break these lines.
Maybe we can also mention how to transfer variables `python<->gdb`. One way is `gdb.parse_and_eval()` but this works one way and is sometimes award to use. There is a better way, using `gdb.convenience_variable()` and `gdb.set_convenience_variable()`.

Reply:
Example:
```
(gdb) p debug::the_database
$1 = (seastar::sharded<replica::database> *) 0x7fffffffbdc0
(gdb) set $db=debug::the_database
(gdb) pi
>>> local_db = sharded(gdb.convenience_variable('db')).local()
>>> gdb.set_convenience_variable('local_db', local_db)
>>> 
(gdb) p $local_db
$2 = (replica::database *) 0x60000575e010
```